cmd/agent/app/flare.go

```diff
@@ -81,12 +90,24 @@ func makeFlare(caseID string) error {
 		logFile = common.DefaultLogFile
 	}
 
-	var filePath string
+	var profile *flare.Profile
 	var err error
+	if profiling >= 30 {
+		fmt.Fprintln(color.Output, color.BlueString("Creating a %ds performance profile.", profiling))
+		profile, err = createPerformanceProfile()
```

Comment:
Should we not use `profiling` somewhere to pull profiling seconds of data ?

Reply:
Yes, in the `cpuProfURL`